orangecontrib/spectroscopy/irfft.py

```diff
@@ -373,22 +347,18 @@ def __call__(self, ifg, zpd=None, phase=None):
         Nzff = self.zff*number_of_points
```

Comment:
This is no longer strictly true, as the zero_fill() function will pad to the nearest power of 2 for DFT efficiency.
I suggest putting Nzff = ifg.shape[0] after the zero_fill() on L357.
Of course check this for your particular calculations.